test.js

```diff
@@ -6,14 +6,20 @@ import processExists from 'process-exists';
 import delay from 'delay';
 import m from './';
 
-// Ensure the noop process has time to exit
-const noopProcessExitDelay = 10;
+async function noopProcessExists(t, expectExists, pid) {
```

Comment:
True, I overcomplicated for no good reason. It's out, whoever needs it can add it back 😄.